bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Page.java

```diff
@@ -238,6 +238,18 @@ default String getTitle() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * If this page is associated with a Template, then this method will return the Template's client libraries categories for the head tag.
+     *
+     * @return an array of client libraries categories; the array can be empty if the page doesn't have an associated template or if the
+     * template has no client libraries
```

Comment:
Would communicate the fact that these are head client libraries in the return doc